In [1]:
import pandas as pd
import numpy as np

---

In [7]:
sample = pd.read_csv('msk_chord_2024/data_clinical_sample.txt', sep='\t', comment='#')
sample['GENE_PANEL'].value_counts()

GENE_PANEL
IMPACT468          12891
IMPACT505           7155
IMPACT410           3973
IMPACT341           1019
IMPACT-HEME-400        2
Name: count, dtype: int64

----

In [8]:
gene_list = pd.read_csv('misc-data/cancerGeneList.tsv', sep = '\t')
gene_list[gene_list['MSK-IMPACT'] == 'Yes'].shape[0]

505

In [10]:
gene_list[gene_list['MSK-HEME'] == 'Yes'].shape[0]

575

this is a 505 gene list, the latest panel. need to find ones that match all panels. 